Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -287,5 +290,13 @@ public ChromiumConfigurationElement Edge
     {
       get { return (ChromiumConfigurationElement)this[_edge]; }
     }
+
+    /// <summary>
+    /// Run the web browser without a user interface (headless mode).
+    /// </summary>
+    public bool Headless
+    {
+      get { return (bool)this[_headless]; }
```

Comment:
The update in WebTestConfigurationSectionTest is missing. This will then also fail the missing schema update.